elements/rh-disclosure/docs/10-style.md

```diff
@@ -0,0 +1,120 @@
+## Style
+
+A disclosure is a container that includes a caret icon and a text label, similar to an [Accordion](/elements/accordion) panel. When expanded, some styles are added for emphasis.
+
+### Anatomy
+
+<figure>
+  <uxdot-example>
+    <img alt="Anatomy of a disclosure component with numbered annotations."
+         src="../disclosure-style-anatomy.svg"
+         width="1012"
+         height="341">
+  </uxdot-example>
+  <figcaption>
+    <ol style="font-size: var(--rh-font-size-body-text-sm, 0.875rem);">
+      <li>Caret icon</li>
+      <li>Title text</li>
+      <li>Container</li>
+      <li>Thick border</li>
+      <li>Box shadow</li>
+    </ol>
+  </figcaption>
+</figure>
+
+## Color scheme
+
+### Light scheme
+
+<uxdot-example>
+  <img alt="Two disclosures. One closed, one open on a light color scheme."
```

Comment:
Down the line we should inline all of `<rh-disclosure>`'s docs SVGs with `light-dark()` and fallback vars (and unique ids)